### ISSUE TITLE
Virtual Environment creation doesn't work on mac os

### ISSUE THREAD

**Issue by bilbo7833** [ahmarz/mentalhealth#1]
A small installation issue:

`virtualenv -p virtualenv_name`
 doesn't work, as the `-p` flag expects a python version. Removing the `-p` flag worked for me on mac os 10.14.6.
`virtualenv virtualenv_name`

**Comment by ahmarz** [comment]
Yes, you are right removing the -p flag will take the default version you have on your mac os, i thought i put in the -p flag if you have different versions of python and interested in using a particualr python version
